css/acquia_lift.admin.css

```diff
@@ -300,6 +300,31 @@ tr.acquia-lift-report-control {
   background-color: #0074BD;
   color: #FFF;
 }
+.acquia-lift-campaign-add .el-card__title {
+  display: block;
+  width: 100%;
+}
+.acquia-lift-campaign-add .el-card .el-card__title:before,
+.acquia-lift-campaign-add .el-card .el-card__title:after {
+  display: inline-block;
+  fill: #fff;
+  width: 1.4rem;
+  height: 1.4rem;
+  line-height: 1em;
+  vertical-align: text-bottom;
+}
+.acquia-lift-campaign-add .el-card .el-card__header .el-card__title[aria-controls]:before {
+  content: url('../images/actions__add.svg');
+}
+.acquia-lift-campaign-add .el-card .el-card__title:after {
+  content: url('../images/actions__close.svg');
```

Comment:
The admin interfaces should be fine with svg only. The audience will not see these images.